src/main/resources/site/parts/husleieCalculator/husleieCalculator.jsx

```diff
@@ -477,9 +493,11 @@ function HusleieCalculator(props) {
             <Button className='submit-one-year' onClick={submitOneYearLater} ref={submitButton}>
               {chooseFiguresToCalculateRent.oneYearLater.phraseOneYearLater}
             </Button>
+            <span className='visually-hidden'>, knapp eller</span>
```

Comment:
Trenger vi å få lest opp ordet knapp? Kanskje det holder med "eller", men da må det være en frase slik at det blir riktig på engelsk.

Reply:
Jeg mener at det hadde gått fint uten (, knapp eller) og (, knapp)

oppgaven ville ha det slikt :
«Det er mer enn ett år siden du endret husleie. Du kan velge hvilke tall som skal brukes til å beregne ny husleie. Mars 1987, knapp eller September 2023 (nyeste tall), knapp»

Er ikke sikker på om det er ett stort krav, at de må være med, Jeg skal høre med Magnus hva han synes (reporter)

Reply:
Added Language check for 'knapp'

    const buttonPhrase = props.language === 'en' ? 'button' : 'knapp'